src/components/networks/createNetworkDialog.jsx

```diff
@@ -215,13 +215,15 @@ const DhcpRow = ({ ipVersion, rangeStart, rangeEnd, expanded, onValueChanged, va
                           onChange={() => onValueChanged('ipv' + ipVersion + 'DhcpEnabled', !expanded)} />
             </FormGroup>
             {expanded && <Grid hasGutter md={6}>
-                <FormGroup fieldId={'create-network-ipv' + ipVersion + '-dhcp-range-start'} label={_("Start")}>
+                <FormGroup fieldId={'create-network-ipv' + ipVersion + '-dhcp-range-start'}
+                           className={`create-network-ipv${ipVersion}-dhcp-range-start`} label={_("Start")}>
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1162-20230808-121812-dac73b38-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)